src/modules/navigator/mission.cpp

```diff
@@ -2091,6 +2104,14 @@ void Mission::replayCachedTriggerItems()
 	}
 }
 
+void Mission::replayCachedSpeedChangeItems()
+{
+	if (_last_speed_change_item.nav_cmd > 0) {
```

Comment:
if this is just about speed commands, shouldnt we just look here for the exact NAV_CMD corresponding to a DO_CHANGE_SPEED?

Also - why are we sending the command again instead of just applying the properties? (sorry im a bit out of the loop on all this)